### ISSUE TITLE
Bots' Healing Priority is non-existent or bad?

### ISSUE THREAD

**Issue by Kaillera** [Regalis11/Barotrauma#2033]
- [Yes] I have searched the issue tracker to check if the issue has already been reported.

**Description**
When bots have multiple people that qualifies for healing, they don't necessarily choose the person in more critical conditions. Even if the critical person is lying flat in front of the bot, they'll run to heal someone else.

**Steps To Reproduce**
Set or tell a bot to heal before/after giving damage to multiple players/bots as well as putting one or two of them into critical.

**Version**
0.9.7
Windows

**Additional information**
Basically you're more likely to die when in critical conditions, the more damage the crew has suffered overall, or not enough time was given to the bot to heal everyone else first.

**Comment by itchyOwl** [comment]
Should be fixed by Regalis11/Barotrauma-development@87e9b2a and Regalis11/Barotrauma-development@dc1a1ff.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
